spec/factories/cfe_submissions.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :cfe_submission, class: CFE::Submission do
+    legal_aid_application { create :legal_aid_application }
+
+    trait :initialised do
+      aasm_state { 'initialised' }
+    end
```

Comment:
I don't think we should be using traits for setting single attributes. They just complicate the factory and obscure the change. That is, in this example:
```ruby
let(:cfe_submission) { create :cfe_submission, aasm_state: :initialized }

let(:cfe_submission) { create :cfe_submission, :initialized }
```
I don't think the saving of 12 characters in the second example, is worth the loss of clarity in the first example.